package.json

```diff
@@ -53,9 +53,21 @@
     "fail": ["@semantic-release/github"]
   },
   "dependencies": {
+    "@material-ui/core": "^1.0.0",
+    "axios": "^0.18.0",
+    "jsona": "^1.1.10",
+    "prop-types": "^15.6.1",
+    "qs": "^6.5.2",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
-    "react-scripts": "1.1.4"
+    "react-redux": "^5.0.7",
+    "react-router": "^4.2.0",
+    "react-router-dom": "^4.2.2",
+    "react-scripts": "1.1.4",
+    "redux": "^4.0.0",
+    "redux-devtools-extension": "^2.13.2",
```

Comment:
Does this make more sense as a dev dep?

Reply:
Good question! This is pulled into the Redux store ([here](https://github.com/EditVR/editvr-frontend/pull/2/files/5514a20761e59351be1c38cfecfeb08d9236dfff#diff-0d73fa95a83c11328f41709eab4e4f82R24)), regardless of whether or not the app is dev or prod.

Reply:
Sweet, makes sense.

Reply:
@techninja I went ahead and imported the dev only devtools in the redux store generator: https://github.com/EditVR/editvr-frontend/pull/2/files#diff-0d73fa95a83c11328f41709eab4e4f82R10